packages/common/src/components/CurrentUserBadge/index.tsx

```diff
@@ -73,8 +58,8 @@ export const CurrentUserBadge = (props: {
         trigger="click"
       >
         <div className="wallet-key" style={walletKeyStyle}>
-          {name && <span style={{ marginRight: '0.5rem' }}>{name}</span>}
-          {image}
+          <span style={{ marginRight: '0.5rem' }}>{wallet.name}</span>
+          <img src={wallet.icon} style={iconStyle} />
```

Comment:
Let's show the wallet address Indenticon and shortened address here. It's more important than the wallet type and name. 
I think there is no need to show the wallet type name at all, but if it looks good we might show the wallet type icon. 

Here is a screenshot from `Governance` design

![image](https://user-images.githubusercontent.com/5817016/129734144-7de52f81-dcad-482e-a77a-13c9e9e916af.png)

Reply:
I was confused by the original code here, because it seems like if there's a wallet icon, it will use that instead of the Identicon. I may have misunderstood the intent, but I think showing the wallet icon and address will be good. I'll rely on you to tell me how useful the Identicon is, since it doesn't seem that helpful to me.

Reply:
Ideally we should have a way to add an avatar to a wallet address and display it here, but without it I would think the Identicon is the next best thing we can do. I personally recognise my wallets using it, but it might be because they are just for different wallets so the wallet type icon would work as well. I don't really have a strong opinion here so we can try to use the wallet type icon instead

Reply:
Fixed by 06089d1e4311dd960e51752affbfb889b4a6e34b. Uses the wallet icon with the abbreviated address.